test/test/util/dash_parser_util.js

```diff
@@ -52,6 +52,8 @@ shaka.test.Dash = class {
     await stream.createSegmentIndex();
 
     shaka.test.ManifestParser.verifySegmentIndex(stream, references);
+
+    dashParser.stop();
```

Comment:
This is a very good candidate for try/finally around everything from dashParser.start to verifySegmentIndex

Reply:
Done!